composer.json

```diff
@@ -1,14 +1,14 @@
 {
     "name": "async-interop/event-loop",
-    "description": "An event loop interface",
+    "description": "An event loop interface for interoperability.",
```

Comment:
I still feel a bit uneasy with that. May you please split that into another PR so that others may agree or disagree on it separately?